calculator/parser.py

```diff
@@ -56,16 +66,49 @@ def push(e, stack):
     stack.append(e)
 
 
-def pop(stack):
+def pop(list, type):
```

Comment:
Please split `pop`, `peek` and `push` into the following (with appropriate docstrings 👀  :
* `stack_pop`
* `stack_push`
* `stack_peek`
* `queue_enqueue`
* `queue_deque`
* `queue_peek`

Reply:
👍